proarc-webapp/src/main/java/cz/cas/lib/proarc/webapp/client/presenter/DigitalObjectEditor.java

```diff
@@ -550,7 +550,7 @@ private EditorDescriptor getDatastreamEditor(DatastreamEditorType type) {
                 break;
             case MEDIA:
                 title = i18n.DigitalObjectEditor_MediaEditor_Title();
-                deditor = new MediaEditor(i18n);
+                deditor = new MediaEditor(i18n, this.getClass().getName());
```

Comment:
Pokud jde o `this.getClass().getName()`, držme se prosím v případě konfigurace raději konstant. O důvodech více [zde](https://github.com/proarc/proarc/pull/562#discussion_r130426415).